spim_registration/tools/luxendo_resave/submit-jobs

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
@schmiedc what is the purpose of this file? We decided to use snakemake to remain agnostic to the scheduler as much as possible. This script appears to drop this.

Reply:
@psteinb this is a preprocessing step that should happen before the snakemake workflow. Its purpose was to resave the microscope specific files of the luxendo system into .tif. These .tif files would then be processed by the snakemake workflow. Since the fileformat is quite oddly organised it would be hard to integrate it otherwise.